### PR TITLE
Chore/wp core plugin updates [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by dpellenwood**
## What does this do/fix?
* Updates WordPress core to v6.0
* Updates misc plugins

HSPH has been running 6.0 during development and didn't come across any issues. Probably good to make sure any new projects use the latest major version.

### CODE REVIEW

File: changelog.md
Comment (defunctl) [inline changelog.md:7]: Newer entries should be at the top so it reads newest updates to oldest :)

**Review by lindseyolson — APPROVED** [review]

**Review by defunctl — APPROVED** [review]
Approving, up to you if you want to make the changelog suggestion but I find it much more logical to scan based on top to bottom.

**Review by Vinsanity — APPROVED** [review]
Should we be waiting for 6.1 to do this?

**Comment by dpellenwood** [comment]
> Should we be waiting for 6.1 to do this?
@Vinsanity , We've been using 6.0 on a variety of pre-launch / in-development projects for a while now. It's safe for new projects. Our general rule of thumb is that *new* projects should start (or update to) the latest version while in development. This allows them to stay up-to-date with Core versions as they progress and launch with the most-recent version of Core available. 

For already launched projects (those in maintenance), yes, we would prefer to wait for the first bug fix release after a major version, 6.0.1 in this case.

The difference is that already live projects have more risk for bugs or regressions not being found during a standard QA/smoketest cycle, and pre-launch projects have more space/time to address any issues since the site isn't live yet.

**Comment by dpellenwood** [comment]
> if a theme is using a theme.json file there are markup changes to the image & video WP blocks which breaks our alignment styles for media in SQ1.

Thanks @jamiepaul ! Good to know! I'll be sure to address that here before merging.

**Review by ChrisMKindred — APPROVED** [review]
👍

**Comment by dpellenwood** [comment]
Closing this as we'll want to wait to update core to 6.0.1 or 6.1 when [the underlying issue](https://core.trac.wordpress.org/ticket/54905) is addressed. And, we'll want to bump plugins before then.

**Pull request closed without merging**
